### PR TITLE
Implement the homology tool

### PR DESCRIPTION
**Pull request by uosl**

### CODE REVIEW

**Comment by uosl** [comment]
Thanks, I'm looking into adding documentation to the code now.

By the way, I didn't test mines that have multiple neighbours, and I doubt this works properly. (If you know of such a mine, please let me know!)

**Comment by yochannah** [comment]
ah - multiple neighbours probably *is* important! 

YeastMine has two neighbourhoods: animals and microbes. This might be a good test case.

**Comment by uosl** [comment]
@yochannah please check before we forget [[=

File: src/index.js [hunk 1]
```diff
@@ -1,26 +1,287 @@
...
+      resolve([symbol, organism]);
     });
+  });
+}
+
+// Use our mine's API URL to get its neighbours.
+function queryNeighbours(mineUrl) {
+  return new Promise(function(resolve) {
+    fetch(
+      "http://registry.intermine.org/service/namespace?url="
```
Comment (yochannah) [inline src/index.js]: let's extract the registry base url to the top of the file to make it easily editable and only used once, and also make it https

**Comment by yochannah** [comment]
thanks for the reminder on this one @uosl!

**Comment by uosl** [comment]
Thanks for the tips! Check my changes in the latest commits.
I thought *Show all in YeastMine (34+)* would be too long so I gave *Open in YeastMine (34+)* a try.

**Pull request merged**